### PR TITLE
updated the insights js file to include all new titles from Summit 2020

### PR DESCRIPTION
**Pull request by dvoss-rh**
Edited and added new sections and titles to correspond to the Summit 2020 release titles.

### CODE REVIEW

**Comment by dvoss-rh** [comment]
@ryelo Any chance you can look this over and help me get this into the beta site if everything is correct?

File: src/Routes.js [hunk 1]
```diff
@@ -28,7 +29,8 @@ export const Routes = (props) => {
     return (
         <Switch>
             <InsightsRoute path={ paths.insights } component={ Insights } rootClass='insights'/>
-            <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/>
+            <InsightsRoute path={ paths.cost } component={ Cost } rootClass='cost'/>
+            #<InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/>
```
Comment (ryelo) [inline src/Routes.js:33]: You have an extra `#` here that needs to be removed

**Review by ryelo — CHANGES_REQUESTED** [review]
Overall it looks good. Just fix that extra character and I can update our config files and we can get this pushed up!

File: src/Routes.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import React from 'react';
 import some from 'lodash/some';
 
 import Insights from './Routes/Insights';
-import CloudManagement from './Routes/CloudManagement';
+import Cost from './Routes/Cost';
+#import CloudManagement from './Routes/CloudManagement';
```
Comment (ryelo) [inline src/Routes.js:8]: Another one here. If you're trying to comment it out, you'll need to do
`// import CloudManagement from './Routes/CloudManagement'; `

**Review by ryelo — CHANGES_REQUESTED** [review]

**Comment by dvoss-rh** [comment]
@ryelo hopefully that's correct. the coffee is not fully kicked in yet.

File: src/Routes.js [hunk 3]
```diff
@@ -28,7 +29,8 @@ export const Routes = (props) => {
     return (
         <Switch>
             <InsightsRoute path={ paths.insights } component={ Insights } rootClass='insights'/>
-            <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/>
+            <InsightsRoute path={ paths.cost } component={ Cost } rootClass='cost'/>
+            //* <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/>
```
Comment (ryelo) [inline src/Routes.js:33]:
```suggestion
            { /* <InsightsRoute path={ paths.cloudManagement } component={ CloudManagement } rootClass='cloudManagement'/> /* }
```

Comment (ryelo) [reply src/Routes.js:33]: It should be this way

**Review by ryelo — CHANGES_REQUESTED** [review]

File: src/Routes.js (hunk 3 above)
Comment (dvoss-rh) [reply src/Routes.js:33]: You are a patient person. I made the change.

Comment (ryelo) [reply src/Routes.js]: You still need the closing `*/ }` at the end of the line

**Comment by dvoss-rh** [comment]
@ryelo hopefully that does the trick.

File: src/Routes.js (hunk 3 above)
Comment (dvoss-rh) [reply src/Routes.js]: ...ootClass='cloudManagement'/> */ } That's what I'm seeing at the end.

**Review by ryelo — CHANGES_REQUESTED** [review]
One last thing `src/Routes/cost.js` should be `src/Routes/Cost.js` (capital "C" in cost)

**Review by ryelo — APPROVED** [review]

**Pull request merged**